internal/dao/generic.go

```diff
@@ -98,14 +98,13 @@ func (g *Generic) Delete(ctx context.Context, path string, propagation *metav1.D
 		return fmt.Errorf("user is not authorized to delete %s", path)
 	}
 
-	const defaultKillGrace = 1
-	var grace int64
-	if force {
-		grace = defaultKillGrace
-	}
 	opts := metav1.DeleteOptions{
-		PropagationPolicy:  propagation,
-		GracePeriodSeconds: &grace,
+		PropagationPolicy: propagation,
+	}
+
+	if force {
+		defaultKillGrace := int64(1)
```

Comment:
Might as well use this instead to make it clearer imho var defaultKillGrace int64 = 1

Reply:
👍  fixed